src/SDK.ts

```diff
@@ -229,8 +229,9 @@ export default class SDK implements ISDK {
     // construct a endpoint for anonymous chats to get LWI Details
     let requestPath = `/${OmnichannelEndpoints.LiveChatLiveWorkItemDetailsPath}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
     const axiosInstance = axios.create();
-    
-    axiosRetry(axiosInstance, { headerOverwrites: [OmnichannelHTTPHeaders.authCodeNonce], 
+
+    axiosRetryHandler(axiosInstance, {
+      headerOverwrites: [OmnichannelHTTPHeaders.authCodeNonce],
```

Comment:
Do we still have parity with `authCodeNonce` headers being sent for auth chats?

Reply:
yes. we are updating headers in axiosRetryHandler function on api failure and following same logic written previously in axiosRetry custom handler function.